framework/src/application/node/block_processor_v0.js

```diff
@@ -1,258 +0,0 @@
-			data => validateSchema(data),
-			({ block }) => this.chainModule.blockReward.calculateReward(block.height),
-			({ block }, expectedReward) =>
-				this.chainModule.validateBlockHeader(
-					block,
-					getBytes(block),
-					expectedReward,
-				),
-		]);
-
-		this.forkStatus.pipe([
-			({ block, lastBlock }) => this.bftModule.forkChoice(block, lastBlock), // validate common block header
-		]);
-
-		this.verify.pipe([
-			({ block, stateStore, skipExistingCheck }) =>
-				this.chainModule.verify(block, stateStore, { skipExistingCheck }),
-		]);
-
-		this.apply.pipe([
-			({ block, stateStore }) => this.chainModule.apply(block, stateStore),
-			({ block, stateStore }) =>
-				this.dposModule.apply(block, stateStore, {
-					delegateListRoundOffset,
-				}),
-		]);
-
-		this.applyGenesis.pipe([
-			({ block, stateStore }) =>
-				this.chainModule.applyGenesis(block, stateStore),
-			({ block, stateStore }) =>
-				this.dposModule.apply(block, stateStore, {
-					delegateListRoundOffset,
-				}),
-		]);
-
-		this.undo.pipe([
-			({ block, stateStore }) => this.chainModule.undo(block, stateStore),
-			({ block, stateStore }) =>
-				this.dposModule.undo(block, stateStore, {
-					delegateListRoundOffset,
-				}),
-		]);
-
-		this.create.pipe([data => this._create(data)]);
-	}
-
-	// eslint-disable-next-line class-methods-use-this
-	get version() {
-		return 0;
-	}
-
-	_create({ transactions, previousBlock, keypair, timestamp }) {
-		const nextHeight = previousBlock ? previousBlock.height + 1 : 1;
-		const reward = this.chainModule.blockReward.calculateReward(nextHeight);
-		let totalFee = BigInt(0);
-		let totalAmount = BigInt(0);
-		let size = 0;
-
-		const trs = transactions || [];
-		const blockTransactions = [];
-		const transactionsBytesArray = [];
-
-		// eslint-disable-next-line no-restricted-syntax
-		for (const transaction of trs) {
-			const transactionBytes = transaction.getBytes(transaction);
-
-			if (size + transactionBytes.length > this.constants.maxPayloadLength) {
-				break;
-			}
-
-			size += transactionBytes.length;
-
-			totalFee += BigInt(transaction.fee);
-			totalAmount += BigInt(transaction.asset.amount || 0);
-
-			blockTransactions.push(transaction);
-			transactionsBytesArray.push(transactionBytes);
-		}
-
-		const transactionsBuffer = Buffer.concat(transactionsBytesArray);
-		const payloadHash = hash(transactionsBuffer).toString('hex');
-
-		const block = {
-			version: this.version,
-			totalAmount,
-			totalFee,
-			height: nextHeight,
-			reward,
-			payloadHash,
-			timestamp,
-			numberOfTransactions: blockTransactions.length,
-			payloadLength: size,
-			previousBlockId: previousBlock ? previousBlock.id : null,
-			generatorPublicKey: keypair.publicKey.toString('hex'),
-			transactions: blockTransactions,
-		};
-
-		return {
-			...block,
-			blockSignature: signDataWithPrivateKey(
-				hash(getBytes(block)),
-				keypair.privateKey,
-			),
-		};
-	}
-}
-
-module.exports = {
-	BlockProcessorV0,
```

Comment:
Why we removed this file? Is it related to this issue?

Reply:
It was removed because it was not used in anywhere, and by changing the function name, it was going to be outdated. Also, with the new regenesis proposal, this is no longer required